pos_session_pay_invoice/readme/INSTALL.rst

```diff
@@ -0,0 +1,2 @@
+* In order to install this module, you will need to install first the
```

Comment:
I think this file is not necessary, as it is redondant with manifest.

Reply:
yes, you are right, I just removed it.